test/integration/targets/aws_waf_web_acl/aliases

```diff
@@ -0,0 +1,5 @@
+cloud/aws
```

Comment:
This needs `posix/ci/cloud/group4/aws` added to run these in CI.

Reply:
Done